src/components/Feedback/Feedback.tsx

```diff
@@ -1,18 +1,47 @@
-import React, { useState } from "react";
+import React, { useContext, useEffect, useState } from "react";
 import { submitFeedback } from "~/src/lib/api/FeedbackApi";
-import { useFeedbackBox } from "@nypl/design-system-react-components";
+import { FeedbackBoxViewType } from "@nypl/design-system-react-components";
+import { FeedbackContext } from "~/src/context/FeedbackContext";
+
+const DEFAULT_DESCRIPTION_TEXT = "Please share your question or feedback.";
+const ERROR_DESCRIPTION_TEXT = "We are here to help!";
 
 const Feedback: React.FC<any> = ({ location }) => {
-  const [view, setView] = useState<"form" | "confirmation" | "error">("form");
-  const { FeedbackBox, isOpen, onOpen, onClose } = useFeedbackBox();
+  const [view, setView] = useState<FeedbackBoxViewType>("form");
+  const [descriptionText, setDescriptionText] = useState(
+    DEFAULT_DESCRIPTION_TEXT
+  );
+  const {
+    FeedbackBox,
+    isOpen,
+    onOpen,
+    onClose,
+    isError,
+    notificationText,
+    setIsError,
+    setNotificationText,
+  } = useContext(FeedbackContext);
+
+  useEffect(() => {
+    if (isError) setDescriptionText(ERROR_DESCRIPTION_TEXT);
+    else setDescriptionText(DEFAULT_DESCRIPTION_TEXT);
+  }, [isError]);
+
+  const onCloseAndReset = () => {
+    if (isError) setIsError(false);
+    if (notificationText) setNotificationText(null);
```

Comment:
Could be simplify this and just setIsError(false) and setNotificationText(null) regardless of previous state? 

Or would that cause a unnecessary rerender?